### PR TITLE
Update 'What is conda?' Intro Page

### PR DESCRIPTION
**Pull request by beeankha**
Connected to issue https://github.com/conda-incubator/conda-dot-org/issues/80

**To Do:**
- [x] Remove "What is conda?" question from FAQ page
- [x] Move "What is conda?" info to home page
- [x] Make the "Learn More" button go directly to FAQ page
- [x] Delete corresponding page from "Learn" section

### CODE REVIEW

**Comment by Kalderon-Sheikhman** [comment]
Hello team, I would very much like to contribute to this Conda repository. Please is this issue resolved or I may draft up a piece for it?

File: src/components/Info/index.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export default function Info() {
+  return (
+    <div className={[styles.info, styles.section_padding].join(' ')}>
+      <div className={styles.info_heading}>
+        <h1 className={styles.gradient_text}>What is conda?</h1>
+        <p>
+        Initially started as a multiplatform package management tool, the term conda has since
+        evolved to encompass an entire <b>open-source packaging ecosystem and philosophy</b>. This ecosystem
+        is supported by many organizations who all share the common goal of <b>providing easier
+        access to programming tools and libraries</b>.
```
Comment (beeankha) [inline src/components/Info/index.jsx:14]: Love this re-write!! 👍

File: learn/faq.md [hunk 2]
```diff
@@ -4,15 +4,6 @@ sidebar_position: 2
...
 
-## What is the difference between Anaconda, Miniconda, and conda?
-
-
-* Anaconda®️ is a Python/R data science distribution that contains conda. Anaconda®️ costs nothing to install and use, [ships with 250 packages](https://docs.anaconda.com/anaconda/packages/pkg-docs/), and offers free community support.
-
-* Miniconda is a small, bootstrap version of Anaconda®️ that includes only conda, Python, the packages that they depend on, and a small number of other popular and useful packages. Users of Miniconda can run the `conda install` command to install 720+ additional conda packages from the Anaconda®️ repository.
-
-* conda is ___
-
```
Comment (beeankha) [inline learn/faq.md:10]: Is it OK if this FAQ question is answered and committed in a future PR?

Comment (beeankha) [reply learn/faq.md:10]: Just filed an issue (and assigned myself):
https://github.com/conda-incubator/conda-dot-org/issues/107

**Pull request merged**